src/yamlfix/entrypoints/__init__.py

```diff
@@ -6,38 +6,49 @@
 
 import logging
 import sys
+from enum import Enum
 
-# Ansi color codes
-RED = 31
-YELLOW = 33
-CYAN = 36
-GREEN = 32
+
+class ANSIFGColorCode(Enum):
+    """ANSI escape codes for colored output."""
+
+    BLACK = 30
+    RED = 31
+    GREEN = 32
+    YELLOW = 33
+    BLUE = 34
+    MAGENTA = 35
+    CYAN = 36
+    WHITE = 37
+    RESET = 0
 
 
 class ConsoleColorFormatter(logging.Formatter):
     """Custom formatter that prints log levels to the console as colored plus signs."""
 
+    # ANSI escape codes for colored output
     colors = {
-        logging.DEBUG: GREEN,
-        logging.INFO: CYAN,
-        logging.WARNING: YELLOW,
-        logging.ERROR: RED,
+        logging.DEBUG: ANSIFGColorCode.GREEN,
+        15: ANSIFGColorCode.BLUE,
+        logging.INFO: ANSIFGColorCode.CYAN,
+        logging.WARNING: ANSIFGColorCode.YELLOW,
+        logging.ERROR: ANSIFGColorCode.RED,
     }
 
     def format(self, record: logging.LogRecord) -> str:
         """Format log records as a colored plus sign followed by the log message."""
-        color = self.colors.get(record.levelno, 0)
-        self._style._fmt = f"[\033[{color}m+\033[0m] %(message)s"  # noqa: W0212
+        color = self.colors.get(record.levelno, ANSIFGColorCode.RESET)
+        self._style._fmt = f"[\033[{color.value}m+\033[0m] %(message)s"  # noqa: W0212
         return super().format(record)
 
 
-def load_logger(verbose: bool = False) -> None:
+def load_logger(verbose: int = 0) -> None:
     """Configure the Logging logger.
 
     Args:
-        verbose: Set the logging level to Debug.
+        verbose: Decrease logging threshold by 10 for each level.
```

Comment:
I find it counterintuitive the use of `decrease` when you're increasing the number of `-v`, I'd rephrase this to `increase the logging level`. Also people might not know that the difference between levels is 10 or 5, so I'd remove that info too